npm/react/src/hooks.ts

```diff
@@ -1,7 +1,7 @@
 // @ts-ignore
 const unfetch = require('unfetch/dist/unfetch.js')
 // @ts-ignore
-const isComponentSpec = () => Cypress.spec.specType === 'component'
+const isComponentSpec = () => true // TODO: refactor code depending on this fn
```

Comment:
Not sure what you mean by this comment

Reply:
specType doesn't exist any more -- this was when we were mixing the mount commands with integration specs. We need to go through and fix specType not being sent through to mobx.